### PR TITLE
vdk-events: improve  Productionizing Jupyter Notebooks README

### PR DESCRIPTION
**Pull request by duyguHsnHsn**
What: Added additional information from  https://github.com/versatile-data-kit-demo/productionizing-jupyter-notebooks/tree/main to the events README

Why:  To make it easier for the people to launch and understand the tutorial  

Signed-off-by: Duygu Hasan [hduygu@vmware.com](mailto:hduygu@vmware.com)

### CODE REVIEW

**Review by antoniivanov — COMMENTED** [review]
remove explore from the name of hte pdf. Also what is this pdf ? We should not have pdf in source control in general.

**Comment by duyguHsnHsn** [comment]
> remove explore from the name of hte pdf. Also what is this pdf ? We should not have pdf in source control in general.

It is the presentation for Explore i can remove it

**Comment by antoniivanov** [comment]
> > remove explore from the name of hte pdf. Also what is this pdf ? We should not have pdf in source control in general.
> 
> It is the presentation for Explore i can remove it

Yeah, we do not need it. Let's embed some of the images in the notebook. 
We want this tutorial to be usable in multiple events

File: events/productionizing-jupyter-notebooks/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+**Congratulations!**
+
+## Feedback
+Please share your feedback :
+[Productionizing Jupyter Notebooks Survey](https://forms.office.com/pages/responsepage.aspx?id=yjiRs-48Skuk1s2D2d1i8AGV0VaygrpPnt7Tz5bBbeBUNFA5NkU3QzlNWEQyUFJCTTQwRUszWk9GUS4u)
+
+Give us a star if you've liked the [project](https://github.com/vmware/versatile-data-kit).
+
+## Where to Find Us
+- [YouTube](https://www.youtube.com/channel/UCasf2Q7X8nF7S4VEmcTHJ0Q/about)
```
Comment (yonitoo) [inline events/productionizing-jupyter-notebooks/README.md:64]: Is there a particular reason why the link leads to the about window and not the channel home instead?

**Review by yonitoo — APPROVED** [review]
LGTM. Good job! Apart from the minor comment I left, a nit (subjective): sometimes "Notebook" is uppercase and sometimes not.

File: events/productionizing-jupyter-notebooks/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+1. **Retrieve Data:** - Extract data from the specified URL using pandas.
+2. **Data Cleansing:**  - Eliminate records associated with 'testuser'.
+3. **Score Classification:** - Assign scores into predefined categories for clarity.
+4. **Data Ingestion:** - Use VDK job_input to ingest the organized data.
+
+### Versatile Data Kit Jupyter Integration
+For detailed instructions on working with VDK, please refer to the guide from the provided [link](../../projects/vdk-plugins/vdk-jupyter/getting-started.ipynb).
+
+## Tutorial
+### [LAUNCH TUTORIAL](https://mybinder.org/v2/gh/versatile-data-kit-demo/productionizing-jupyter-notebooks/HEAD?labpath=tutorial-job%2F10_notebook.ipynb)
```
Comment (antoniivanov) [inline events/productionizing-jupyter-notebooks/README.md:43]: Provide  backup link with https://notebooks.gesis.org/binder/

File: events/productionizing-jupyter-notebooks/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+* Understood the process of ingesting data through VDK within a Jupyter environment.
+* Understood the process of making notebooks ready for production with VDK.
+
+**Congratulations!**
+
+## Feedback
+Please share your feedback :
+[Productionizing Jupyter Notebooks Survey](https://forms.office.com/pages/responsepage.aspx?id=yjiRs-48Skuk1s2D2d1i8AGV0VaygrpPnt7Tz5bBbeBUNFA5NkU3QzlNWEQyUFJCTTQwRUszWk9GUS4u)
+
+Give us a star if you've liked the [project](https://github.com/vmware/versatile-data-kit).
```
Comment (antoniivanov) [inline events/productionizing-jupyter-notebooks/README.md:61]: I don't like to encourage people to give stars, so remove it.

File: events/productionizing-jupyter-notebooks/README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- Discuss strategies for productionizing Jupyter Notebooks.
+- Showcase hands-on examples of these challenges in the Jupyter UI.
+
+## Purpose
+The purpose of this scenario is to demonstrate how to operationalize Jupyter notebooks using the Versatile Data Kit (VDK) Jupyter integration. By the end of this guide, you'll understand how to:
+* Create a data job with VDK within a Jupyter notebook.
+* Write a data workflow in a notebook and make it ready to be put in a production environment.
+
+## Background
+### Objective:
```
Comment (antoniivanov) [inline events/productionizing-jupyter-notebooks/README.md:32]: Again those are not the objective. I feel we discuss this a few times. 

This is the setup of the tutorial. The tutorial walks you through standard ETL process  retrieve, clean, transform, load data and addresses the challenges often people in notebooks have while doing these things.

**Review by antoniivanov — COMMENTED** [review]
Fix the description of the PR. It's not updated

File: events/productionizing-jupyter-notebooks/README.md (hunk 1 above)
Comment (duyguHsnHsn) [reply events/productionizing-jupyter-notebooks/README.md:64]: @yonitoo i have copied it from the other events README, so i suppose it was decided so back in the time

File: events/productionizing-jupyter-notebooks/README.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+### Versatile Data Kit Jupyter Integration
+For detailed instructions on working with VDK, please refer to the guide from the provided [link](../../projects/vdk-plugins/vdk-jupyter/getting-started.ipynb).
+
+## Tutorial
+### **🌟🚀 [CLICK HERE TO LAUNCH THE TUTORIAL!](https://mybinder.org/v2/gh/versatile-data-kit-demo/productionizing-jupyter-notebooks/HEAD?labpath=tutorial-job%2F10_notebook.ipynb)🚀🌟**
+
+The link did not work?
+Try this one out:
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/versatile-data-kit-demo/productionizing-jupyter-notebooks/HEAD?labpath=tutorial-job%2F10_notebook.ipynb)
```
Comment (antoniivanov) [inline events/productionizing-jupyter-notebooks/README.md:47]: That link is the same as the above one ?!
```
https://notebooks.gesis.org/binder/v2/gh/versatile-data-kit-demo/productionizing-jupyter-notebooks/HEAD?labpath=tutorial-job%2F10_notebook.ipynb

**Review by antoniivanov — APPROVED** [review]

**Pull request merged**